x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/Driver.java

```diff
@@ -7,6 +7,8 @@
 
 package org.elasticsearch.compute.operator;
 
+import org.apache.logging.log4j.LogManager;
```

Comment:
Can you back out changes in this class?

Reply:
Wait but I thought the changes in this class are those that allow us to stop the processing? If I remove these changes, what will be stopping the processing then?